nntrainer/src/util_func.cpp

```diff
@@ -76,27 +76,28 @@ Tensor softmax(Tensor t) {
   divisor.setZero();
 
   for (int k = 0; k < batch; k++) {
-    int index = k * height;
+    int index = k * height * width;
+    float m = -std::numeric_limits<float>::max();
```

Comment:
```suggestion
    float m = std::numeric_limits<float>::lowest();
```
Regarding `std::numeric_limits<float>::lowest != -std::numeric_limits<float>::max` it might be not equal depending on machine
https://stackoverflow.com/questions/42405800/numeric-limits-of-float-lowest-and-max

Reply:
fixed.